lib/pages/setup/route_list.dart

```diff
@@ -53,89 +54,167 @@ class _RouteListState extends State<RouteList> {
 
     setState(() {
       widget.routeParams.routes = routes;
-      print('## ${routes.length} routes found');
-
-      widget.routeParams.routes.forEach((r) => routeList.add(RouteListEntry(
-            r.title,
-            r.date,
-            r.totalLength,
-          )));
-      
+      widget.routeParams.routes.forEach((r) => _routeList.add(RouteListEntry(r)));
+      if(routes.length > 0) _title = routes[0].title;
       this._routesCalculated = true;
     });
 
   }
 
+  // TODO add localization or remove if not needed
+  headerText() {
+    String paramTitles = 'Start: ';
+    (_title.length > 20) ? paramTitles += '\n\nDistance: ' : paramTitles += '\nDistance: ';
+    if(widget.routeParams.poiCategories.length > 0) {
+      paramTitles += '\nPOIs: ';
+      for(var i = 1; i < widget.routeParams.poiCategories.length; i++) paramTitles += '\n';
+      // widget.routeParams.poiCategories.forEach((p) => paramTitles += '\n');
+    }
+    paramTitles += '\nAltitude differences: ';
+
+    String params = '$_title';
+    params += '\n${widget.routeParams.distanceKm.toInt()} KM / ${(widget.routeParams.distanceKm*12).toInt()} MIN';
```

Comment:
Why is KM and MIN alway written in caps?
The symbol for kilometre is _km_ and the symbol for minute is _min_.
https://en.wikipedia.org/wiki/Kilometre
https://en.wikipedia.org/wiki/Minute